cmd/config.go

```diff
@@ -58,6 +58,12 @@ func init() {
 	_ = viper.BindPFlag("git.exclude_list", configCmd.PersistentFlags().Lookup("exclude_list"))
 	_ = viper.BindPFlag("git.template_file", configCmd.PersistentFlags().Lookup("template_file"))
 	_ = viper.BindPFlag("git.template_string", configCmd.PersistentFlags().Lookup("template_string"))
+	_ = viper.BindPFlag("git.api_base_url", configCmd.PersistentFlags().Lookup("api_base_url"))
+	_ = viper.BindPFlag("git.token", configCmd.PersistentFlags().Lookup("token"))
+	_ = viper.BindPFlag("git.owner", configCmd.PersistentFlags().Lookup("owner"))
+	_ = viper.BindPFlag("git.repo", configCmd.PersistentFlags().Lookup("repo"))
+	_ = viper.BindPFlag("server.ip", configCmd.PersistentFlags().Lookup("ip"))
+	_ = viper.BindPFlag("server.port", configCmd.PersistentFlags().Lookup("port"))
 }
 
 var configCmd = &cobra.Command{
```

Comment:
Error generating comment for file cmd/config.go: Post "https://api.openai.com/v1/chat/completions": proxyconnect tcp: dial tcp 127.0.0.1:10838: connectex: No connection could be made because the target machine actively refused it.